app/src/main/java/nextstep/signup/model/Password.kt

```diff
@@ -0,0 +1,29 @@
+package nextstep.signup.model
+
+data class Password(val password: String) {
+    fun isError() = !isBlank() && !isValid()
+
+    fun isValid() = isValidLength() && hasEnglishAndNumber()
+
+    fun getErrorMessage(): String? =
+        when {
+            isBlank() -> null
+            !isValidLength() -> PASSWORD_LENGTH_ERROR
+            !hasEnglishAndNumber() -> PASSWORD_FORM_ERROR
+            else -> null
+        }
+
+    private fun isBlank(): Boolean = password.isBlank()
+
+    private fun isValidLength(): Boolean = password.length in PASSWORD_MIN_LENGTH..PASSWORD_MAX_LENGTH
+
+    private fun hasEnglishAndNumber(): Boolean = password.matches(Regex(PASSWORD_REGEX))
+
+    companion object {
+        private const val PASSWORD_REGEX = "^(?=.*[a-zA-Z])(?=.*[0-9]).{8,16}$"
```

Comment:
```suggestion
        private const val PASSWORD_REGEX = "^(?=.*[a-zA-Z])(?=.*[0-9])$"
```
`isValidLength` 에서 길이 검사를 해주기때문에 `.{8,16}` 는 불필요할 것 같아요!

Reply:
정규 표현식에 `.{8,16}`가 포함되어 있는 걸 놓쳤네요! 감사합니다~

`반영 커밋` : [99bd67b](https://github.com/woowacourse/android-signup/pull/40/commits/99bd67b70a4d37266cd57c2a7030a71899722209)